src/v/config/configuration.cc

```diff
@@ -496,12 +496,13 @@ configuration::configuration()
   , target_quota_byte_rate(
       *this,
       "target_quota_byte_rate",
-      "Target request size quota byte rate (bytes per second) - 2GB default",
+      "Target request size quota byte rate (bytes per second) - disabled "
+      "default (= 0)",
       {.needs_restart = needs_restart::no,
        .example = "1073741824",
        .visibility = visibility::user},
       target_produce_quota_byte_rate_default,
-      {.min = 1_MiB})
+      {.min = 0})
```

Comment:
Is it worth adding a validator to only allow `0, >1MiB`?

Reply:
I don't think so. The 1MiB is arbitrary, and we don't have it for fetch nor for quotas configured through the Kafka API, so in my opinion it's better not to have it.